eventing/README.md

```diff
@@ -191,6 +191,21 @@ The GcpPubSubSource fires a new event each time a message is published on a
   [ObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectreference-v1-core)
   A reference to the object that should receive events.
 
+### AwsSqsSource
+
+The AwsSqsSource fires a new event each time an event is published on an
+[AWS SQS topic](https://aws.amazon.com/sqs/).
+
+**Spec fields**:
+
+- `queueURL`: URL of the SQS queue to pull events from.
+- `awsCredsSecret`: credential to use to poll the AWS SQS queue.
+- `sink`:
+  [ObjectReference](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#objectreference-v1-core)
+  A reference to the object that should receive events.
+- `serviceAccountName`: `string` The name of the ServiceAccount used to
+  access the `awsCredsSecret`.
```

Comment:
Might want to clarify that the Receive Adapter runs as this account. So, the effect is that we access the secret as this Service Account, but it's because the Receive Adapter runs as this Service Account.

Maybe add something like this (not verbatim, but shrug)

	// serviceAccountName is the name of the ServiceAccount that will be used to
	// run the Receive Adapter Deployment.

Reply:
This seems to parallel line 184 for the GcpPubSubSource.